crates/fs/Cargo.toml

```diff
@@ -24,8 +24,10 @@ libc.workspace = true
 parking_lot.workspace = true
 paths.workspace = true
 rope.workspace = true
+rpc.workspace = true
```

Comment:
Taking a dependency on `rpc` in `fs` seems a little fishy, to me.

Reply:
Ah good point, made it a proto dep instead!